client/uploads.go

```diff
@@ -274,6 +294,23 @@ func (u *MultiUpload) getChildrenIDs() []string {
 	return ids
 }
 
+func (u *MultiUpload) changeProgress(newBytesTransferred int64) {
+	var notify bool
+
+	u.mutex.Lock()
+	defer u.mutex.Unlock()
+	if u.totalSizeBytes > 0 {
+		u.totalBytesTransferred += newBytesTransferred
+		newProgress := int((100 * float64(u.totalBytesTransferred)) / float64(u.totalSizeBytes))
+		notify = newProgress != u.status.Progress
+		u.status.Progress = newProgress
+	}
```

Comment:
Do we need else here? It should cover the error scenario when there is no total but some transferred bytes are reported.